import { HttpException, HttpStatus } from '@nestjs/common';

export const apontamentoNaoEncontradoException = () =>
  new HttpException(
    { error: 'Apontamento n√£o encontrado.' },
    HttpStatus.NOT_FOUND,
  );
